nixos/lib/test-driver/default.nix

```
@@ -46,4 +46,6 @@ python3Packages.buildPythonApplication {
     echo -e "\x1b[32m## run black\x1b[0m"
     black --check --diff .
   '';
+
+  passthru.tests = nixosTests.nixos-test-driver.driver;
```

I was thinking:

```suggestion
  passthru.tests = nixosTests.nixos-test-driver;
```

(Resuming thread https://github.com/NixOS/nixpkgs/pull/157161#discussion_r1369010401)

---

*Ah*, why not, actually.